Source/WebKit/UIProcess/Cocoa/NavigationState.mm

```diff
@@ -514,6 +514,10 @@ static void tryInterceptNavigation(Ref<API::NavigationAction>&& navigationAction
 #if ENABLE(WK_WEB_EXTENSIONS)
 static bool isUnsupportedWebExtensionNavigation(API::NavigationAction& navigationAction, WebPageProxy& page)
 {
+    bool subframeNavigation = navigationAction.targetFrame() && !navigationAction.targetFrame()->isMainFrame();
+    if (subframeNavigation)
+        return false;
```

Comment:
I think it might be slightly easier just to pass `subframeNavigation` from both of the methods that call this, instead of computing it again.

Reply:
I think I prefer this, so it is all enclosed. These are inline methods anyway, so it is fast.

Reply:
Even for a bool? Interesting!

I retract my comment. I thought if the way we compute `subframeNavigation` ever changed, it would be better to pass it, but that probably won't happen.